schemas/session-replay/mobile/webview-wireframe-schema.json

```diff
@@ -9,17 +9,17 @@
       "$ref": "_common-shape-wireframe-schema.json"
     },
     {
-      "required": ["type", "nestedEnvId"],
+      "required": ["type", "slotId"],
       "properties": {
         "type": {
           "type": "string",
           "description": "The type of the wireframe.",
           "const": "webview",
           "readOnly": true
         },
-        "nestedEnvId": {
-          "type": "integer",
-          "description": "Defines the unique ID of the replayed webview environment that will be nested in this container.",
+        "slotId": {
+          "type": "string",
```

Comment:
Discussed with @mariusc83 string are more reliable that integer (cf. int64 non supported long number in javascript issue)